<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VistaProgreso</class>
 <widget class="QMainWindow" name="VistaProgreso">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>POWERGYM - Progreso de Entrenamiento</string> </property>
  <property name="styleSheet"> <string notr="true">
    QMainWindow#VistaProgreso { /* Usamos el objectName de la ventana principal */
        background-color: #f0f0f0; /* Color de fondo gris claro */
    }
    QWidget#centralwidget {
        background-color: #f8f8f8; /* Color de fondo del widget central, un poco más claro */
    }
    
    QLabel { /* Estilo general para todos los QLabel */
        color: #333333;
        font-size: 14px;
        font-weight: 500;
    }

    QLabel#labelTitulo { /* Estilo específico para el título principal "Progreso de tus ejercicios" */
        color: #B71C1C; /* Rojo oscuro para el texto del título */
        font-size: 24px; /* Tamaño de fuente grande */
        font-weight: bold;
        padding-bottom: 10px;
    }

    QPushButton {
        background-color: #D32F2F; /* Rojo vibrante */
        color: white;
        border: none;
        border-radius: 15px; /* Bordes más redondeados */
        padding: 10px 20px; /* Padding para botones */
        font-size: 16px;
        font-weight: bold;
        min-height: 40px; /* Altura mínima para botones */
    }
    QPushButton:hover {
        background-color: #C62828; /* Rojo más oscuro al pasar el ratón */
    }

    QComboBox {
        background-color: white;
        border: 2px solid #D32F2F; /* Borde rojo */
        border-radius: 8px;
        padding: 5px;
        color: black;
        font-size: 14px;
    }
    QComboBox::drop-down {
        border-left: 1px solid #C62828;
        width: 25px;
    }
    QComboBox::down-arrow {
        image: url(:/icons/arrow_down.png); /* Asegúrate de tener un icono de flecha en recursos.qrc */
        width: 16px;
        height: 16px;
    }
    QComboBox:focus {
        border: 2px solid #C62828;
    }

    /* Estilo para el widget que contiene la gráfica */
    QWidget#graficaWidget {
        background-color: white; /* Fondo blanco para la gráfica */
        border: 1px solid #E0E0E0;
        border-radius: 8px;
    }

   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btn_volver">
      <property name="text">
       <string>Volver al menú</string>
      </property>
     </widget>
    </item>

    <item>
     <widget class="QLabel" name="labelTitulo">
      <property name="text">
       <string>Progreso de tus ejercicios</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>

    <item>
     <widget class="QComboBox" name="comboEjercicio">
      <property name="currentText">
       <string>Sentadilla</string>
      </property>
      <item>
       <property name="text">
        <string>Sentadilla</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Press Banca</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Peso Muerto</string>
       </property>
      </item>
     </widget>
    </item>

    <item>
     <widget class="QWidget" name="graficaWidget">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>400</height>
       </size>
      </property>
     </widget>
    </item>

   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../recursos.qrc"/>
 </resources>
 <connections/>
</ui>